Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/AuditTransformDataBuilder.java

```diff
@@ -0,0 +1,194 @@
+ *     * Redistributions of source code must retain the above
+ *       copyright notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the documentation
+ *       and/or other materials provided with the distribution.
+ *     * Neither the name of the United States Government nor the
+ *       names of its contributors may be used to endorse or promote products
+ *       derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE UNITED STATES GOVERNMENT BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package gov.hhs.fha.nhinc.audit;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/*
+ *
+ * @author achidamb
+ */
+public class AuditTransformDataBuilder {
+
+    public Map<String, String> eventIdCodeMap;
+    public Map<String, String> eventCodeSystemMap;
+    public Map<String, String> eventDisplayNameRequestorMap;
+    public Map<String, String> eventDisplayNameResponderMap;
+    public Map<String, String> eventTypeCodeMap;
+    public Map<String, String> eventTypeCodeSystemMap;
+    public Map<String, String> eventTypeCodeDisplayNameMap;
+    public Map<String, String> eventActionCodeRequestorMap;
+    public Map<String, String> eventActionCodeResponderMap;
+
+    private AuditTransformDataBuilder() {
+        eventIdCodeMap = createEventIdCodeMap();
+        eventCodeSystemMap = createEventCodeSystemMap();
+        eventDisplayNameRequestorMap = createEventDisplayNameRequestorMap();
+        eventDisplayNameResponderMap = createEventDisplayNameResponderMap();
+        eventTypeCodeMap = createEventTypeCodeMap();
+        eventTypeCodeSystemMap = createEventTypeCodeSystemMap();
+        eventTypeCodeDisplayNameMap = createEventTypeCodeDisplayNameMap();
+        eventActionCodeRequestorMap = createEventActionCodeRequestorMap();
+        eventActionCodeResponderMap = createEventActionCodeResponderMap();
+    }
+
+    private Map<String, String> createEventIdCodeMap() {
+        eventIdCodeMap = new HashMap();
+        eventIdCodeMap.put("PatientDiscovery", "110112");
+        return eventIdCodeMap;
```

Comment:
I don't think this is the right way to do this.  I think you probably should create a hashmap per service and put that map in the service's lib.

for example, you'd have a class in PD service jar that has the map:

Map<String,String> patientDiscoveryAuditMap = new HashMap<>():

private void loadMap() {
     patientDiscoveryAuditMap.put("eventDisplayName", "PatientDiscovery");
     patientDiscoveryAuditMap.put("eventTypeCode", "ITI-55");
     ...
}